### PR TITLE
Fix#99

### PR DESCRIPTION
**Pull request by anjali-rathod**
1. Added content of all books issued in admindashboard
2. Minor changes in contact us page

### CODE REVIEW

File: public-html/asset/scss/components/contact.scss [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    position: absolute;
+    left: 0;
+    padding: 5px 0px;
+    font-size: 16px;
+    margin:10px 0px;
+    pointer-events: none;
+    transition: 0.5s;
+    color:$contactblack;
+}
+.contactcontainer .contactform .contactinputbox input:focus ~ span,textarea:focus ~ span {
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/contact.scss]: one selector per line

**Review by rahulsavaria — CHANGES_REQUESTED** [review]

**Comment by rahulsavaria** [comment]
why we multiple CSS files there should be only one CSS file

File: public-html/asset/scss/components/contact.scss [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    background-image:url("../images/drupal2 (copy).png");
+    background-repeat: no-repeat;
+    height: 100%;
+    position: relative;
+    padding: 0px 150px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+ }
+.contactus .contactcontent {
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/contact.scss]: use nested format for selectors

**Review by rahulsavaria — CHANGES_REQUESTED** [review]

File: public-html/asset/scss/components/contact.scss (hunk 1 above)
Comment (anjali-rathod) [reply public-html/asset/scss/components/contact.scss]: done

**Review by guptahemant — CHANGES_REQUESTED** [review]
Resolve conflicts

**Pull request closed without merging**